### ISSUE TITLE
jquery-confirm modal does not close after external function call

### ISSUE THREAD

**Issue by steamngn** [craftpip/jquery-confirm#592]
**jquery-confirm version:**
v3.3.4

**I'm submitting a ...**  (check one with "x")
[ ] bug report
[ ] feature request
[X] support request

**Current behavior:**
I have a confirm modal that has a form loaded using detach and append. The form has a submit button in it that calls an ajax submit.
All of this works fine except the jquery-confirm modal does not close when this submit button is pressed. Just not sure how to handle this - is there a method to close the modal from this button press or do i need to somehow incorporate the submit into another button in the modal?

**Expected behavior:**
The jquery-confirm modal should close on either the submit button press or the ajax success return called by the button. the jquery-confirm button code:

**Steps to reproduce:**

**Related code:**
```
buttons:{
                ACCEPT:{
                btnClass:'allowancebtn',
                text:'Accept Damage Allowance',
                action: function(accept){
                $("#cust_refund_amt").val(v.target.dataset.allowance);//this will be sell price(prc) for refund, allowance(allowance) for damage allowance
                signeeded = 1;
             var elem = $("#canvas").detach();//.appendTo("#parentNode");
             $(elem).insertBefore('.damages .damagesend');
             var elem = $("#damages").detach();//.appendTo("#parentNode");
             this.setContentAppend(elem);
              var cw = $("#canvas").innerWidth();
             var co = $("#canvas").offset();
             zkSignature.capture(cw,co);
            $.each(this.buttons, function (key, val) {
            this.hide();
    });
    return false;
                }
                },
                DECLINE:{
                btnClass:'damagebtn',
                text:'Decline Allowance',
                action: function(decline){
                
                }
                }
                }
```

**Other information:**

**Comment by steamngn** [comment]
added another button to the confirm to act as submit. Problem solved.
